charts/airbyte-cron/templates/deployment.yaml

```diff
@@ -85,21 +80,11 @@ spec:
               configMapKeyRef:
                 name: {{ .Release.Name }}-airbyte-env
                 key: CRON_MICRONAUT_ENVIRONMENTS
-          - name: REMOTE_CONNECTOR_CATALOG_URL
-            valueFrom:
-              configMapKeyRef:
-                name: {{ .Release.Name }}-airbyte-env
-                key: REMOTE_CONNECTOR_CATALOG_URL
           - name: TRACKING_STRATEGY
             valueFrom:
               configMapKeyRef:
                 name: {{ .Release.Name }}-airbyte-env
                 key: TRACKING_STRATEGY
-          - name: UPDATE_DEFINITIONS_CRON_ENABLED
-            valueFrom:
-              configMapKeyRef:
-                name: {{ .Release.Name }}-airbyte-env
-                key: UPDATE_DEFINITIONS_CRON_ENABLED
```

Comment:
Not sure about airbyte_role, but for these two (remote_connector_catalog_url and update_definitions_cron_enabled), while they _could_ be set on OSS, they're mainly meant for cloud usage so it's probably ok to remove for now if it's causing an issue.

Reply:
@perangel is there a reason not to add them to the configmap?

Reply:
I see that `AIRBYTE_ROLE` is declared in https://github.com/airbytehq/airbyte/blob/master/airbyte-cron/src/main/resources/application.yml#L69. Anything not declared by the ConfigMap can still be provided via `env_vars` or `global.env_vars` so there's still a means to pass them through. But since these keys are definitely **not** in the ConfigMap, it should be safe to remove them.

Reply:
As for me, I've never seen those variables present in OSS deployments, only in cloud one, so if devs are ok with dropping those(meaning that it'll be safe), then we can drop them and test the deployment out.
For this you need to run the following:

```shell
mv charts/airbyte/Chart.yaml charts/airbyte/Chart.yaml.old
mv charts/airbyte/Chart.yaml.test charts/airbyte/Chart.yaml 
cd charts/airbyte && helm dep update && cd -
helm upgrade --install --debug --values charts/airbyte/values.yaml.test airbyte charts/airbyte

```